pages/vi/profiles/johnbeatrice.md

```diff
@@ -0,0 +1,15 @@
+# John Beatrice
+
+Location | Timezone | OS
+:--- | :--- | :---
+Greater Boston | EST | Ubuntu 18.04.3|
+
+## About Me
+Hi there, I'm a self-taught web developer transitioning careers, and I'm very excited to be a part of OLE! I've worked in a variety of industries before now but I feel I've found a great fit in tech and am super excited to work as a software engineer!
+
+
+## Contact
+* __LinkedIn:__ https://www.linkedin.com/in/john-beatrice-2071a8218/
```

Comment:
please use markdown for the link. Right now it is browser which is showing it as link not the markdown